src/_pages-content/health-and-safety.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Health and Safety Guidelines
+layout: content.liquid
+permalink: /health-and-safety/
+---
+### WHY WE (STILL) HAVE A HEALTH & SAFETY POLICY?
+
+We are nice people and many of us and our fellow community members can’t attend without health and safety guidelines in place. We want PyCon India to be an event that everyone feels safe attending. PyCon India is exceptional in that we have people attending from all over the world—last year we had people attending from 75 different countries! We love that people bring their unique perspectives… but they also bring their local germs and Covid strains. We’re masking to keep our immuno-compromised friends safe at PyCon India and to ensure that those of us who share their homes with immuno-compromised people can also consider joining us without quarantining when they return home.
+
+We are going to be doing some analysis on the airflow and refresh rates while we are in the convention center this year to see if there are some spaces that are well-ventilated enough to be indistinguishable from an outdoor situation for next PyCon India 2025. But for 2024, we are sticking with masking in all of the conference spaces.
+
+### MASKS ARE REQUIRED
```

Comment:
"Encouraged"

Please modify the rest content accordingly.

Reply:
please give me the info to add..

Reply:
We don't enforce if someone is wearing masks or not. It's advised to wear.